---
title: ObservableMultipleTest.Catch_ErrorSpecific_Caught Method  (ReactiveTests.Tests)
TOCTitle: Catch_ErrorSpecific_Caught Method
ms:assetid: M:ReactiveTests.Tests.ObservableMultipleTest.Catch_ErrorSpecific_Caught
ms:mtpsurl: https://msdn.microsoft.com/en-us/library/reactivetests.tests.observablemultipletest.catch_errorspecific_caught(v=VS.103)
ms:contentKeyID: 36619795
ms.date: 06/28/2011
mtps_version: v=VS.103
f1_keywords:
- ReactiveTests.Tests.ObservableMultipleTest.Catch_ErrorSpecific_Caught
dev_langs:
- CSharp
- JScript
- VB
- FSharp
- c++
---

# ObservableMultipleTest.Catch\_ErrorSpecific\_Caught Method

**Namespace:**  [ReactiveTests.Tests](hh289046\(v=vs.103\).md)  
**Assembly:**  Tests.System.Reactive (in Tests.System.Reactive.dll)

## Syntax

``` vb
'Declaration
<TestMethodAttribute> _
Public Sub Catch_ErrorSpecific_Caught
```

``` vb
'Usage
Dim instance As ObservableMultipleTest

instance.Catch_ErrorSpecific_Caught()
```

``` csharp
[TestMethodAttribute]
public void Catch_ErrorSpecific_Caught()
```

``` c++
[TestMethodAttribute]
public:
void Catch_ErrorSpecific_Caught()
```

``` fsharp
[<TestMethodAttribute>]
member Catch_ErrorSpecific_Caught : unit -> unit 
```

``` jscript
public function Catch_ErrorSpecific_Caught()
```

## See Also

#### Reference

[ObservableMultipleTest Class](hh303586\(v=vs.103\).md)

[ReactiveTests.Tests Namespace](hh289046\(v=vs.103\).md)

